src/Sulu/Bundle/MediaBundle/Media/ImageConverter/ImagineImageConverter.php

```diff
@@ -189,9 +201,13 @@ public function convert(FileVersion $fileVersion, $formatKey, $imageFormat)
 
         $image->strip();
 
-        // Set Interlacing to plane for smaller image size.
-        if (1 == count($image->layers())) {
-            $image->interlace(ImageInterface::INTERLACE_PLANE);
+        try {
+            // Set Interlacing to plane for smaller image size.
+            if (1 == count($image->layers())) {
+                $image->interlace(ImageInterface::INTERLACE_PLANE);
+            }
+        } catch (\Exception $exception) {
```

Comment:
Isn't there a better exception to catch?

Reply:
@wachterjohannes change to atleast [Imagine\Exception\RuntimeException](https://github.com/contao/imagine-svg/blob/b62cae7d217c7d11efd88b5c2f373f87814767f8/src/Image.php#L15)